docs/enterprise/metrics-app.md

```diff
@@ -26,7 +26,7 @@ To install the app follow the instructions in [Install the Microsoft Fabric Capa
 
 When using the Microsoft Fabric Capacity Metrics app, consider the following limitations.
 
-* In the [CU over time](metrics-app-overview-page.md#cu-over-time) visual logarithmic's view, the primary axis seen on the left of the visual, isn't aligned with the secondary axis seen on the right of the visual.
+* In the [Capacity utilization and throttling](metrics-app-overview-page.md#capacity-utilization-and-throttling) visual logarithmic's view, the primary axis seen on the left of the visual, isn't aligned with the secondary axis seen on the right of the visual.
```

Comment:
@Balaji-Shewale can we remove the comment about data warehouse only showing OneLake?  This was fixed as of 9/18.

Reply:
Removed !